app/routes.py

```diff
@@ -54,22 +55,24 @@ def logout():
     return redirect(url_for('all_complaints'))
 
 
-def basic_data(query, column_list):
+def basic_data(query, column_list): # Zbyt generyczna nazwa. Nic mi ona nie mówi.
     # search filter
+    # Cała ta funkcja jest zdecydowanie za duża i trudna w czytaniu, nie wspominając o zrozumieniu tego co tu się dzieje.
     search = request.args.get('search[value]')
     if search:
         query = query.filter(db.or_(
             InCom.description.like(f'%{search}%'),
             InCom.order_number.like(f'%{search}%')
         ))
-    total_filtered = query.count()
+    total_filtered = query.count() # redundante, lepiej użyć query.count() tam gdzie go potrzebujesz, zamiast tworzyć dodatkową zmienną
```

Comment:
Niestety nie, ponieważ query podlega dalej kolejnym zmianom i  jak robię query.count() przy return to nie pokazuje kolejnych części tabeli.